### ISSUE TITLE
Build issue when using gatsby-plugin-netlify-cms

### ISSUE THREAD

**Issue by acarl** [gatsbyjs/gatsby#4226]
I'm receiving the following error on the production but not the develop build when using gatsby-plugin-netlify-cms. I was using this plugin before but on an older version of gatsby. After I upgraded gatsby (among other things) I am getting this error. If I remove gatsby-plugin-netlify-cms I no longer get the error.

```
Error: app-ffa06f381625a48fea6d.js from UglifyJs
  SyntaxError: Unexpected character '`' [./~/gatsby-plugin-netlify-cms/gatsby-br  owser.js:10,0]
```

Could this potentially be related to this recent pull request?
https://github.com/gatsbyjs/gatsby/pull/4215

**Comment by Flaque** [comment]
Also getting this

**Comment by martynhoyer** [comment]
I mentioned this issue within https://github.com/gatsbyjs/gatsby/issues/4227#issuecomment-368295263

There's a PR open: 
https://github.com/gatsbyjs/gatsby/pull/4233

**Comment by cpilson** [comment]
If you absolutely must get past this error, and aren't using Travis-CI, you can replace ```` ` ```` in the file with `'`

The file is located at `node_modules/gatsby-plugin-netlify-cms/gatsby-browser.js`

**Comment by erquhart** [comment]
Fixed in #4233 (pending release).
